test.rb

```diff
@@ -0,0 +1,16 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+
+aaa;
+
+not 1
+
+ l = lambda {|a| a }
+ l.(x)
+ l[x][x].(x)
```

Comment:
__[development][rubocop]__
* Inconsistent indentation detected.
* Prefer the use of `lambda.call(...)` over `lambda.(...)`. [:ref](https://github.com/bbatsov/ruby-style-guide#proc-call)

<!-- Comment by [SideCI](https://sideci.com) -->